src/StackExchange.Redis/Interfaces/IServer.cs

```diff
@@ -33,15 +33,27 @@ public partial interface IServer : IRedis
         bool IsConnected { get; }
 
         /// <summary>
-        /// Gets whether the connected server is a replica / slave
+        /// Gets whether the connected server is a replica
         /// </summary>
+        [Obsolete(Messages.PreferReplica)]
         bool IsSlave { get; }
 
         /// <summary>
-        /// Explicitly opt in for slave writes on writable slaves
+        /// Gets whether the connected server is a replica
         /// </summary>
+        bool IsReplica { get; }
+
+        /// <summary>
+        /// Explicitly opt in for replica writes on writable replica
+        /// </summary>
+        [Obsolete(Messages.PreferReplica)]
         bool AllowSlaveWrites { get; set; }
 
+        /// <summary>
+        /// Explicitly opt in for replica writes on writable replica
+        /// </summary>
+        bool AllowReplicaWrites { get; set; }
```

Comment:
How concerned are we about the interface break here?

Reply:
Note: we break the interface _no matter what_ here - so what's the thinking on best approach? I'm absolutely willing to take a major version here, but it's a nontrivial break regardless - not sure what's the best course of action

Reply:
if by "interface break" you mean "adding things", I'm not in the least bit concerned by that

Reply:
Yep, exactly that - good with me :+1: